app/controllers/api/integrations_controller.rb

```diff
@@ -24,7 +24,7 @@ def update
     if @service.using_proxy_pro? && !@proxy.apicast_configuration_driven
       proxy_pro_update
     elsif @proxy.save_and_deploy(proxy_params)
-      flash[:notice] = flash_message(:update_success)
+      flash[:notice] = flash_message(:update_success, environment: @proxy.service_mesh_integration? ? :Production : :Staging)
```

Comment:
Why the capital letter in `:Production` and `:Staging`? If it is because the goal of this is only showing it to the user and not any logic/code/calculation after it, then why not a String directly? 🤔

Reply:
Right it should be a `String`